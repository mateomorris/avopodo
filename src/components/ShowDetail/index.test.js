import React from 'react';
import { Text } from 'react-native';
import { ShowDetail } from './index';
import renderer from 'react-test-renderer';

it('renders ShowDetail correctly', () => {
  const tree = renderer
    .create(
        <ShowDetail 
          title={'Test Title'}
          image={'https://d3sv2eduhewoas.cloudfront.net/channel/image/4ffedff25d474163b1a8511a6a26b76a.jpg'}
          description={'This is the test description'}
          color={'blue'}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
