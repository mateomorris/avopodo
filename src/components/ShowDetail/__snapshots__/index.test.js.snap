// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ShowDetail correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "black",
      "flexDirection": "row",
      "opacity": 1,
      "paddingBottom": 10,
      "paddingLeft": 10,
      "paddingRight": 10,
      "paddingTop": 10,
      "shadowColor": "black",
      "shadowOffset": Object {
        "height": 1,
        "width": 0,
      },
      "shadowOpacity": 0.8,
      "shadowRadius": 2,
      "with": "30%",
    }
  }
>
  <View
    style={
      Object {
        "margin": 5,
        "shadowColor": "#000000",
        "shadowOffset": Object {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0.75,
        "shadowRadius": 2,
      }
    }
  >
    <Image
      source={
        Object {
          "cache": "force-cache",
          "uri": "https://d3sv2eduhewoas.cloudfront.net/channel/image/4ffedff25d474163b1a8511a6a26b76a.jpg",
        }
      }
      style={
        Array [
          Object {
            "borderRadius": 4,
            "height": 120,
            "width": 120,
          },
          Object {
            "backgroundColor": "blue",
          },
        ]
      }
    />
  </View>
  <View
    style={
      Object {
        "paddingLeft": 10,
        "width": "65%",
      }
    }
  >
    <View
      style={
        Object {
          "height": 130,
        }
      }
    >
      <Text
        ellipsizeMode="tail"
        numberOfLines={7}
        style={
          Object {
            "color": "#fefefe",
            "fontSize": 15,
            "fontWeight": "500",
          }
        }
      >
        This is the test description
      </Text>
    </View>
  </View>
</View>
`;
